packages/block-library/src/navigation/index.php

```diff
@@ -5,6 +5,11 @@
  * @package WordPress
  */
 
+if ( ! class_exists( 'WP_Navigation_Block_Renderer' ) ) {
```

Comment:
I'm not entirely sure that this check is needed. In fact, I think this will create issues after the backport. The file will be using the renamed class but it won't be loaded.

Reply:
this check is also updated in the built version, so I think it's ok